html5lib/html5parser.py

```diff
@@ -201,6 +188,9 @@ def mainLoop(self):
         DoctypeToken = tokenTypes["Doctype"]
         ParseErrorToken = tokenTypes["ParseError"]
 
+        type_names = {value: key for key, value in tokenTypes.items()}
+        debug = self.debug
```

Comment:
@gsnedders Is there a reason to bring `tokenTypes` and `self.debug` into local method variables here?

At [L225](https://github.com/html5lib/html5lib-python/pull/272/files#diff-88cdf3a2fd8f807dae741251459c88e1e10ed54666f13ab7fdc6f4d47ae08648R225) we could compare using `if self.debug` and similarly at [L226](https://github.com/html5lib/html5lib-python/pull/272/files#diff-88cdf3a2fd8f807dae741251459c88e1e10ed54666f13ab7fdc6f4d47ae08648R226) we could use `info = {"type": tokenTypes[type]}`.

(am guessing it's possibly an artifact of some IDE-based refactoring?  Hopefully a straightforward cleanup either way)